### PR TITLE
Update dependencies & Android dependencies

### PR DESCRIPTION
**Pull request by rock3r**
This small PR updates the Flutter and Android dependencies for the app. It does not update Firebase on the Android side because of breaking changes in Firebase 12.0.x which aren't yet compatible with the latest Flutter Firestore plugin.

Note: the changes in the XCode files are not generated by me, they reappear every time I build the app. Seems like something changed in Cocoapods or XCode that causes it.

### CODE REVIEW

File: lib/src/feeds.dart [hunk 1]
```diff
@@ -308,7 +308,7 @@ class Feed implements Comparable<Feed> {
 }
 
 class FeedManager {
-  final http.Client httpClient = new http.Client();
```
Comment (devoncarew) [inline lib/src/feeds.dart]: For now, can you leave in the `new` keyword?

Comment (rock3r) [reply lib/src/feeds.dart]: Sure!

Comment (rock3r) [reply lib/src/feeds.dart]: 79e0c9d

**Comment by devoncarew** [comment]
lgtm, thanks!

**Pull request merged**